### DISCUSSION TITLE
Server Actions doesn't work at producation

### DISCUSSION THREAD

**Discussion by beforedisappear** [vercel/next.js#63840] [Help]
### Summary

I use 2 server action to change cookie and 1 to sent form. All these server actions works perfect at local, but at prod stand i have 500 error and  this uncaught promise : "Uncaught (in promise) Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error."

*prod stand is dockerized app at host (not vercel).

**Comment by kombos** [comment]
can you provide the code for it
